standalone-metastore/pom.xml

```diff
@@ -531,6 +531,29 @@
         </os>
       </activation>
     </profile>
+    <profile>
+      <id>javadoc</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-javadoc-plugin</artifactId>
```

Comment:
Since the javadoc generation is a big mess ATM, I would suggest to keep it as it is, and if the tests are failing then we can decide what we want to do with them.

See also: #3185